src/raspi/io/pneumatics/pressure.py

```diff
@@ -0,0 +1,26 @@
+class Pressure():
+    """
+    Represents air pressure readings to measure current flexion/extension
+    (knee) movement of each leg.
+    """
+    MIN_PRESSURE : float = 0.0  # TODO: assign this
+    MAX_PRESSURE : float = 0.0  # and this
+    
+    pressure : float = 0.0
+    pressure_history : list[float] = []
+
+    def __init__(self) -> None:
+        ...
+
+    def is_sufficient_pressure(self) -> bool:
+        """
+        TODO: this one
+        """
+        return False
+    
+    def get_average_pressure(self) -> float:
```

Comment:
Especially don't know this

Reply:
Removed the Pressure class from the RPi code. Since it now only sees a TRUE/FALSE for whether or not the pressure is good (and this dictates whether other behaviors can run), this could be a global variable somewhere that a main loop can write to every "cycle".